lib/charms/loki_k8s/v1/loki_push_api.py

```diff
@@ -349,6 +352,72 @@ def _promtail_error(self, event):
     )
 ```
 
+## LogForwarder Library Usage
+
+Let's say that we have a workload charm that produces logs to the standard output (stdout),
+and we need to send those logs to a workload implementing the `loki_push_api` interface,
```

Comment:
```suggestion
and we need to send those logs to a Loki instance.
```

if you like, you can add something like: "to know how to reach a loki instance, a charm typically would use the `loki_push_api` interface".